src/renderer/containers/Onboard/SigningKey/Verify/SigningMnemonicQuestion.tsx

```diff
@@ -1,28 +1,38 @@
 import React, {Component, ReactElement} from "react";
 import {VerifyMnemonic} from "../../../../components/VerifyMnemonic/VerifyMnemonic";
 import {connect} from "react-redux";
+import {bindActionCreators, Dispatch} from "redux";
 import {RouteComponentProps} from "react-router";
 import {getRandomInt, getRandomIntArray} from "../../../../services/mnemonic/utils/random";
 import {IRootState} from "../../../../reducers";
+import {storeFailedVerificationAction} from "../../../../actions";
 
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
-interface IProps extends Pick<RouteComponentProps, "history"> {
+interface IOwnProps extends Pick<RouteComponentProps, "history"> {
 
 }
+interface IInjectedProps {
+    storeFailedVerification: typeof storeFailedVerificationAction;
+}
 
-class SigningMnemonicQuestion extends Component<IProps &  Pick<IRootState, "register">, {}> {
+class SigningMnemonicQuestion extends Component<IOwnProps & IInjectedProps &  Pick<IRootState, "register">, {}> {
     public render(): ReactElement {
         const mnemonic = this.props.register.mnemonic.split(" ");
         const randArray = getRandomIntArray(12);
         const correctAnswerIndex = randArray[getRandomInt(3)];
 
+        const handleInvalidAnswer = (): void => {
+            this.props.storeFailedVerification(true);
+            this.props.history.goBack();
+        };
+        
         return (
             <VerifyMnemonic
                 question={`What’s the ${correctAnswerIndex + 1}th word in the mnemonic?`}
                 answers={[mnemonic[randArray[0]], mnemonic[randArray[1]], mnemonic[randArray[2]]]}
                 correctAnswer={mnemonic[correctAnswerIndex]}
                 onCorrectAnswer={(): void => {}}
```

Comment:
I don't think you have latest changes to master, this should navigate somewhere and clear signing mnemonic verification status